### ISSUE TITLE
Mouse scrolling does not work

### ISSUE THREAD

**Issue by hisenbug** [texstudio-org/texstudio#1279]
Mouse scrolling is not working (reopen #623).

## Environment
- TeXstudio: 2.12.15 or higher
- Qt: qt5
- OS: Windows 10
- TeX distribution: MixTex 2.9

### Expected behavior
On a Dell 4K laptop primary screen, the mouse wheel or trackpad scrolls the text up or down.

### Actual behavior
The mouse wheel is unresponsive, scrolling does not work.

Switching the TeXstudio window to an external display makes scrolling work again.

### How to reproduce
Use TeXstudio on a Dell 4K screen (I have Dell XPS 13 7390).

**Comment by JStatnekov** [comment]
This problem appears for me after upgrading to TeXstudio 4.2.1 but only after I do a Find. Initially the mouse wheel scrolls normally but after doing ctrl-F, scrolling no longer works. I do not have Dell 4K screens and the only way that I've found to restore scrolling is to close and re-open TexStudio.

I'm on Manjaro Linux with KDE Plasma 5.23.5 and Qt 5.15.2

**Comment by wujastyk** [comment]
Periodically, the mouse scroll control stops moving text on the TeXstudio screen.  Closing and reopening TS clears the issue. 

The scroll control works fine in all other apps (at the same time that it doesn't work in TS).



TeXstudio 4.8.7 (git 0.0.0++)
Using Qt Version 6.4.2, compiled with Qt 6.4.2 R

System:
  Kernel: 6.8.0-60-generic arch: x86_64 bits: 64 compiler: gcc v: 13.3.0 clocksource: tsc
  Desktop: Cinnamon v: 6.4.8 tk: GTK v: 3.24.41 wm: Muffin v: 6.4.1 vt: 7 dm: LightDM v: 1.30.0
    Distro: Linux Mint 22.1 Xia base: Ubuntu 24.04 noble

Graphics:
  Device-1: Intel Raptor Lake-P [Iris Xe Graphics] driver: i915 v: kernel arch: Gen-13 ports:
    active: HDMI-A-1,HDMI-A-2 empty: DP-1, DP-2, DP-3, DP-4 bus-ID: 00:02.0 chip-ID: 8086:a7a0
    class-ID: 0300

Microsoft bluetooth mouse.

**Comment by sunderme** [comment]
works on ubuntu 24.04 (unity)

**Comment by wujastyk** [comment]
> works on ubuntu 24.04 (unity)

Yes, that's what I'm using.  Ubuntu 24.04 (noble) is the base of Mint 22.1.  Maybe this is a difference between the Gnome and Cinnamon windowing systems?  But it must be recondite because I do not see the effect in any other application.

**Comment by dbitouze** [comment]
Maybe a important information is whether you're on Wayland or not.

**Comment by wujastyk** [comment]
> Maybe a important information is whether you're on Wayland or not.

Not.

**Comment by sunderme** [comment]
I had issuea with mouse scroll on lxqt with firefox
The resolution was switching to unity/wayland
txs only uses qt libraries, there is nothing we ca don about that.

**Comment by wujastyk** [comment]
> I had issuea with mouse scroll on lxqt with firefox The resolution was switching to unity/wayland txs only uses qt libraries, there is nothing we ca don about that.

Yesterday I switched from 4.8.7 to `texstudio-4.8.8beta4-x86_64.AppImage` and so far the scrolling issue has not reappeared.

**Comment by wujastyk** [comment]
TeXstudio 4.8.9 (git 0.0.0++)
Using Qt Version 6.4.2, compiled with Qt 6.4.2 R
(The deb installation, not the AppImage)

Unfortunately, the mouse scrolling freeze is still an issue.  It seems to happen after the editor window has been inactive for a while (like, I go for coffee and come back).  

Also, when I select a menu, the wrong menu opens and it becomes impossible to select an item from the desired menu. Everything seems to happen an inch to the left of the cursor (I click "macros" and "wizards" opens).

Best,
Dominik

**Comment by sunderme** [comment]
That sounds rather like qt/desktop env issue. There is not really anything we can do about that.

**Comment by wujastyk** [comment]
> That sounds rather like qt/desktop env issue. There is not really anything we can do about that.

Okay, thanks.  I'll see if there are some local QT tweaks that might help.  I'll report back if I find an answer.
